docs/src/Atmos/Model/atmosmodel.md

```diff
@@ -0,0 +1,217 @@
+
+### State Variables
+```@docs
+CLIMA.Atmos.vars_state
+```
+
+
+```math
+\vec{Y}=\left( \begin{array}{c}
+\rho \\
+\rho\vec{u} \\
+\rho e^{\mathrm{tot}}\\
+\rho q_k\\
+\rho q_{p,i}\\
+\rho \chi_j
+\end{array}
+\right).
+```
+
+### Fluxes
+
+#### Nondiffusive Fluxes
+```@docs
+CLIMA.Atmos.flux_nondiffusive!
+```
+ ```math
+ \mathrm{F}_{ndiff}=\left( \begin{array}{c}
+ \rho \vec{u} \\
+ \rho \vec{u} \otimes \vec{u} + (p - p_r) \vec{I}_3 \\
+ \rho e^{\mathrm{tot}} \vec{u} + p \vec{u}\\
+ \rho q_k \vec{u}\\
+ \rho q_{p,i} \vec{u} \\
+ \rho \chi_j \vec{u}
+\end{array}
+\right).
+```
+#### Diffusive Fluxes
+```@docs
+CLIMA.Atmos.flux_diffusive!
+```
+```math
+\mathrm{F}_{diff}=\left( \begin{array}{c}
+\rho\vec{d}_{q_t} \\
+\rho\vec{\tau} + \rho\vec{d}_{q_t} \otimes \vec{u}\\
+\vec{u} \cdot \rho\vec{\tau} + \rho (\vec{J} + \vec{D}) \\
+\rho\vec{d}_{q_k}\\
+\rho \vec{d}_{q_{p, i}}\\
+\rho \vec{d}_{\chi_j}
+\end{array}
+\right).
+```
+
+#### Radiation Fluxes
+```math
+\mathrm{F}_{rad} = 
+\left( \begin{array}{c}
+\vec{0} \\
+\vec{0} \\
+\rho \vec{F}_R \\
+\vec{0} \\
+\vec{0} \\
+\vec{0} 
+\end{array}
+\right)
+```
+
+### Fluxes of precipitating species
+```math
+\mathrm{F}_{fall} =
+- \left( \begin{array}{c}
+\rho q_{c} w_{c} \vec{\hat k}  \\
+q_c w_c \vec{\hat k} \otimes \rho \vec{u}  \\
+\rho W_c \vec{\hat k} \\
+\rho q_{k} w_{k} \vec{\hat k}  \\
+\rho q_{p,i} w_{p, i} \vec{\hat k} \\
+\rho \chi_{i} w_{\chi, i} \vec{\hat k}
+\end{array} \right)
+```
+
+#### Sources
+```@docs
+CLIMA.Atmos.source!
+```
+
+```math
+\mathrm{S}(\vec{Y}, \nabla\vec{Y})= 
+ \left( \begin{array}{c}
+ -\rho C(q_t \rightarrow q_p) \\
+  -(\rho - \rho_r) \nabla\Phi - 2 \vec{\Omega} \times \rho\vec{u}  + \rho \vec{F}_{\vec{u}} \\
+ \rho Q - \sum_{j\in\{v,l,i\}} (I_j + \Phi)  \rho C(q_j \rightarrow q_p) - M \\% + \rho \vec{u} \cdot \vec{F}_{\vec{u}}  \\
+\rho C(q_p \rightarrow q_k) + \rho \sum_j \rho C(q_j \rightarrow q_k) \\
+    \rho \sum_k C(q_k \rightarrow q_{p, i}) - \rho \sum_j C(q_{p, i} \rightarrow q_{p, j})\\
+\rho \mathcal{S}_{\chi_i}
+\end{array}
+\right)
+```
+
+## Configurations
+The struct `AtmosModel` defines a specific subtype of a balance law (i.e. conservation equations) specific to 
+atmospheric modelling. A complete description of a `model` is provided by the fields listed below. In this 
+implementation of the `AtmosModel` we concern ourselves with the conservative form of the compressible equations
+of moist fluid motion given a set of initial, boundary and forcing(source) conditions. 
+
+```@docs
+CLIMA.Atmos.AtmosModel
+```
+
+### LES Configuration (with defaults)
+Default field values for the LES `AtmosModel` definition are included below. Users are directed to the 
+model subcomponent pages to view the possible options for each subcomponent.
+```
+    ::Type{AtmosLESConfigType},
+    param_set::AbstractParameterSet;
+    orientation::O = FlatOrientation(),
+    ref_state::RS = HydrostaticState(
+        LinearTemperatureProfile(
```

Comment:
Please replace this by the smooth temperature profile in the reference state. We’ll remove the linear profile soon.

Reply:
I'll link it to the source file so the docs always reflect the latest `default` settings.